orchagent/timer.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
```

Comment:
I prefer move to pragma since it is much more clean and less easy mistaken. It is nearly a practical 'standard'.

[https://en.wikipedia.org/wiki/Pragma_once](https://en.wikipedia.org/wiki/Pragma_once)

---
In reply to: [150723282](https://github.com/Azure/sonic-swss/pull/382#discussion_r150723282) [](ancestors = 150723282)

Reply:
pragma has its advantage.. I was just mentioning on the consistency with other header files